src/components/labels.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+
+interface LabelProps {
+  hideText: string;
```

Comment:
When I read `hideText` feels like it should be receiving a boolean. Can we renamed to something like hideTextStyle or hideTextClass? Or can we add the following 

```
    hideText: {
      overflow: "hidden",
      textOverflow: "ellipsis",
      whiteSpace: "nowrap",
      maxWidth: "10rem",
    }
```

to the component default style with the option to overwrite it?

Reply:
I agree, `hideText` is passed in from various components all using the same values. There are some ways we can improve on it

1. as suggested, have a default style and have a `className` (or similar) prop to overwrite/append it.
2. in the case where we want to have multiple pre-defined styles, we can have a `variant` prop similar to how `Typography` does it.
3. If we want to use custom styling for a particular label, we can export the styles in the module so we can render it in storybook without resorting to hardcoded values.